test/test_utils/__init__.py

```diff
@@ -29,8 +29,26 @@
 DEFAULT_REGION = "us-west-2"
 # Constant to represent region where p3dn tests can be run
 P3DN_REGION = "us-east-1"
+def get_ami_id_boto3(region_name, ami_name_pattern):
+    """
+    For a given region and ami name pattern, return the latest ami-id
+    """
+    ami_list = boto3.client("ec2", region_name=region_name).describe_images(
+        Filters=[
+            {
+                "Name": "name",
+                "Values": [ami_name_pattern],
+            },
+        ],
+        Owners=[
+            'amazon',
+        ],
+    )
```

Comment:
You can reduce the 10 lines of code here to just 3 lines
```suggestion
    ami_list = boto3.client("ec2", region_name=region_name).describe_images(
        Filters=[{"Name": "name", "Values": [ami_name_pattern]}], Owners=['amazon']
    )
```

Reply:
Adding the suggestions and making a commit.